flixkit/fcl_bindings.go

```diff
@@ -104,6 +114,8 @@ type FclGenerator struct {
 	Templates []string
 }
 
+var _ GenerateBinding = (*FclGenerator)(nil)
```

Comment:
This naming feels like an action (like a method) as opposed to a struct that that is returned.

Reply:
very similiar comment i made for @bthaile privately, we should stay consistent, whichever form we take on